.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea
```

Comment:
Do you mind removing this? I'd like to keep editor specific ignores out of here, you can use a global gitignore for this.